modules/content/index.js

```diff
@@ -0,0 +1,192 @@
+require('vue-content-loader')
+const join = require('path').join
+const fs = require('fs')
+const fm = require('front-matter')
+const pathToRegexp = require('path-to-regexp')
+const paramCase = require('param-case')
+
+module.exports = function (options) {
+  const nuxt = this.options
+
+  // convert markdown files to vue component
+  nuxt.build.loaders.push({
+    test: /\.md$/,
+    use: [{
+      loader: 'vue-content-loader',
+      options: options.loaderOptions || {}
+    }]
+  })
+
+  // create content config
+  const config = {}
+  config.options = options
+  config.options.rootPath = join(nuxt.rootDir, options.srcDir || "/")
+  config.content = nuxt.content || options.content || ['/']
+  if (!is2DArray(config.content)) config.content = [config.content]
+
+  // create routes for registered content
+  nuxt.router.extendRoutes = (routes, resolve) => {
```

Comment:
Instead of overwrite `extendRoutes` you may prefer using `this.extendRoutes` [source](https://github.com/nuxt/nuxt.js/blob/v1.0.0-alpha.4/lib/module.js#L80)